policyengine_us/parameters/gov/states/dc/tax/income/credits/kccatc/income_limit.yaml

```diff
@@ -16,6 +16,8 @@ metadata:
       href: https://otr.cfo.dc.gov/sites/default/files/dc/sites/otr/publication/attachments/2022_D-40_Booklet_Final_blk_01_23_23_Ordc.pdf#page=60
```

Comment:
Could you move the uprating to the general metadata here and use the `propagate_metadata_to_children` tag